tests/__init__.py

```diff
@@ -11,44 +11,52 @@ def _compile(compile_func, source):
 
 def _exec(compile_func):
     """Factory to create an execute function."""
+
     def _exec(source, glb=None):
         code = _compile(compile_func, source)
         if glb is None:
             glb = {}
-        exec(code, glb)
+        exec (code, glb)
```

Comment:
Function calls do not have a tailing space in Python. Even if it is a statement in Python 2 we added this parenthesis on purpose to support Python 3.